api/server/server.go

```diff
@@ -1640,8 +1640,9 @@ func allocateDaemonPort(addr string) error {
 		return fmt.Errorf("failed to lookup %s address in host specification", host)
 	}
 
+	pa := portallocator.Get()
```

Comment:
@stevvooe We are hiding the loading behind the `Get`. So to clarify you seem to be opposed to naming this method `Get` If it's called Default() would that be ok? The file io that is happening is on `/proc/sys/net/ipv4/ip_local_port_range` which is a global read only file to read the local port range configured in the system. So this is supposed to be global despite that fact that file io is happening.